### PR TITLE
ADD Metadata to data filtering plugins

### LINKED ISSUE #204: Data filtering plugins should apply to metadata as well
Current data filtering plugins (timestamp compression) do not apply to metadata, but they should process the metadata attributes in the same way as the regular ones.

### PR DESCRIPTION
**Pull request by dmoranj**
Fix #204 

(The branch was reused from an older branch by mistake. Shame on me).

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -4,3 +4,4 @@
 - Modify the answer for the GET devices and GET groups operations (#196).
 - Add APPEND Mode for Context Broker updates.
 - Add Timestamp compression plugin.
+- Add Metadata filtering to data filtering plugins (#204).
```
Comment (gtorodelvalle) [inline CHANGES_NEXT_RELEASE]: Final blank line appreciated... :)

Comment (AlvaroVega) [inline CHANGES_NEXT_RELEASE]: missing new line here

File: test/unit/compress-timestamp-plugin_test.js [hunk 2]
```diff
@@ -200,4 +243,6 @@ describe('Timestamp compression plugin', function() {
             });
         });
     });
+
+
```
Comment (gtorodelvalle) [inline test/unit/compress-timestamp-plugin_test.js]: Maybe you would also like to remove one of these two blank lines... :)

**Comment by XavierVal** [comment]
LGTM

**Pull request merged**